jetstream/tests/test_bigquery_client.py

```diff
@@ -46,7 +46,6 @@ def test_delete_experiment_tables():
         calls,
         any_order=True,
     )
-    assert delete_call.call_count == len(calls)
```

Comment:
What's the reasoning for removing these assertions?

Reply:
Added them back. The way BigQuery results are mocked here is different. `tables_matching_regex` is called multiple times when deleting a table, but the result is now always the same (since the `client.query` result is now mocked to something static)